tests/sample/sigal.conf.py

```diff
@@ -17,7 +17,16 @@
     'sigal.plugins.nomedia',
     'sigal.plugins.watermark',
     'sigal.plugins.zip_gallery',
+    'sigal.plugins.encrypt'
 ]
+encrypt_options = {
+    'password': 'password',
+    'ask_password': True,
+    'gcm_tag': 'AuTheNTiCatIoNtAG',
+    'kdf_salt': 'saltysaltsweetysweet',
+    'kdf_iters': 10000,
+    'encrypt_symlinked_originals': False
+}
```

Comment:
Maybe not add this by default, since this config is used by several tests.

Reply:
I'm not quite familiar with the testing infrastructure. So the plugin options should be hardcoded in the test files? Should this plugin be enabled in the test config by default?

Reply:
Reverted in 28f187f.